test/apps/iodemo/io_demo.cc

```diff
@@ -952,6 +948,10 @@ static int parse_args(int argc, char **argv, options_t *test_opts)
         }
     }
 
+    if (test_opts->chunk_size == 0) {
+        test_opts->chunk_size = test_opts->max_data_size;
+    }
```

Comment:
:+1:

Reply:
std::numeric_limits<unsigned>::max() is used as default chunk size in final commit.